sample-app/src/main/java/com/squareup/cycler/sampleapp/GridMutationsPage.kt

```diff
@@ -0,0 +1,63 @@
+package com.squareup.cycler.sampleapp
+
+import android.widget.Toast
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.squareup.cycler.Recycler
+import com.squareup.cycler.enableMutations
+import com.squareup.cycler.sampleapp.BaseItem.Item
+
+/**
+ * What makes this page different from [MutationsPage] is that
+ * 1. we set a [GridLayoutManager] on the [RecyclerView],
```

Comment:
I wonder if there is a way to "bundle" these four flags into a single `enableGridDragAndDrop` flag.